### PR TITLE
Update of Active Pool Map change related on 28/3

### PR DESCRIPTION
**Pull request by TandelK**
Vertigo in for Active Duty Map and Cache has been moved to reserved. Just keeping the map in Maplist but modified current map pool.

### CODE REVIEW

File: instance/prod_config.py.default [hunk 1]
```diff
@@ -32,11 +32,12 @@ MAPLIST = [
     'de_train',
     'de_dust2',
     'de_season',
+    'de_vertigo',
 ]
 
 # Maps whose checkbox is selected (in the mappool) by default in the "create a match" page
 DEFAULT_MAPLIST = [
-    'de_cache',
+    'de_vertigo',
```
Comment (splewis) [inline instance/prod_config.py.default]: Can this be kept alphabetized?

File: instance/prod_config.py.default [hunk 2]
```diff
@@ -32,11 +32,12 @@ MAPLIST = [
     'de_train',
     'de_dust2',
     'de_season',
+    'de_vertigo',
```
Comment (splewis) [inline instance/prod_config.py.default]: Can this be ordered so the first 7 entries are the same as in DEFAULT_MAPLIST, then it's the remaining ones after?

**Comment by TandelK** [comment]
@splewis Updated. Moved Cache and Cobblestone in Maplist to last as they are no longer part of Active Duty Map Pool. Also updated Default Maplist as per Alphabetic Order.

File: instance/prod_config.py.default (hunk 1 above)
Comment (TandelK) [reply instance/prod_config.py.default]: updated in new commit

**Comment by Dieterfu** [comment]
Please also update it on the public site: get5.splewis.net

**Comment by splewis** [comment]
This project is already just intended to be a proof of concept; my get5.splewis.net is even less reliable in that it's just a live example.

You really should **not** rely on it working, much less being up to date.

**Pull request merged**